packages/core/src/components/button/test/secondary/button.e2e.ts

```diff
@@ -0,0 +1,99 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/button/test/secondary/index.html';
+
+test.describe('tds-button', () => {
+  test('renders secondary button correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    await expect(button).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('component receives click event', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    const myEventSpy = await page.spyOnEvent('click');
+    await button.click();
+    expect(myEventSpy).toHaveReceivedEvent();
+  });
+
+  test('component should have correct background color on hovering', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    const innerButton = button.locator('button');
+    await innerButton.hover();
+    const buttonBackgroundColor = await innerButton.evaluate(
+      (button2) => getComputedStyle(button2).backgroundColor,
+    );
+    expect(buttonBackgroundColor).toBe('rgb(32, 88, 168)');
+  });
+
+  test('component should have correct background color on pressed', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    const innerButton = button.locator('button');
+    await innerButton.hover();
+    await page.mouse.down({ button: 'left' });
+    const buttonBackgroundColor = await innerButton.evaluate(
+      (button2) => getComputedStyle(button2).backgroundColor,
+    );
+    expect(buttonBackgroundColor).toBe('rgba(0, 0, 0, 0)');
+  });
+
+  test('component should have correct border when active', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    const innerButton = button.locator('button');
+    await innerButton.focus();
+    const buttonBackgroundColor = await innerButton.evaluate(
+      (button2) => getComputedStyle(button2).backgroundColor,
+    );
+    const buttonBorderColor = await innerButton.evaluate(
+      (button2) => getComputedStyle(button2).borderColor,
+    );
+    const buttonOutlineColor = await innerButton.evaluate(
+      (button2) => getComputedStyle(button2).outlineColor,
+    );
+    expect(buttonBackgroundColor).toBe('rgba(0, 0, 0, 0)');
+    expect(buttonBorderColor).toBe('rgb(43, 112, 211)');
+    expect(buttonOutlineColor).toBe('rgb(43, 112, 211)');
+  });
+
+  test('Type = Button', async ({ page }) => {
+    await page.goto(componentTestPath);
+    const button = page.locator('tds-button');
+    const innerButton = button.locator('button');
+    const type = await innerButton.getAttribute('type');
+    expect(type).toBe('button');
+  });
+
+  test('check if the inner button contain classes primary lg and sc-tds-button', async ({
```

Comment:
Should this say it's looking for classes secondary and md?

Reply:
[687ce4d](https://github.com/scania-digital-design-system/tegel/pull/433/commits/687ce4dc6710ba31502b8d5f53cfff6c64ffdc44) Updated